### ISSUE TITLE
Aqara Mini Switch not working

### ISSUE THREAD

**Issue by kiihmoo** [mdaskalov/homebridge-tasmota-zbbridge#32]
Hi, 
i have an Sonoff Zbbridge flashed with Tasmota and connected an Aqara WXKG11LM like this one

https://zigbee.blakadder.com/Aqara_WXKG11LM.html

Zigbee join did work, and connection to mosquito seems to be working too.

But I can't get those MQTT Data to work with my other devices like a like bulb

My goal is to push a button and switch on a light bulb

**Logs:**

[16.12.2022, 20:55:53] [tasmota_E32163] Restoring ZbBridge accessory: Sensor3 (0x00158D0008747677) - sensor
[16.12.2022, 20:55:53] [tasmota_E32163] Cleanup - configured 1, cached 1 accessories.

**Plugin Config:**

{
            "name": "tasmota_E32163",
            "mqttTopic": "tasmota_E32163",
            "zbBridgeDevices": [
                {
                    "addr": "0x00158D0008747677",
                    "type": "sensor",
                    "name": "Sensor3",
                    "sensorService": "StatelessProgrammableSwitch",
                    "sensorValuePath": "Click"
                }
            ],
            "platform": "TasmotaZbBridge"
        }

**Screenshots:**
<img width="1543" alt="Bildschirm­foto 2022-12-16 um 21 11 50" src="https://user-images.githubusercontent.com/19505564/208182101-33fe2302-0c7b-43c0-b9bd-6f6cf5704d13.png">

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 1.6.0
* **Node.js Version**: v14.21.1
* **NPM Version**:9.2.0
* **Operating System**: Debian /

**Comment by kiihmoo** [comment]
{"ZbReceived":{"0xD394":{"Device":"0xD394","Name":"Sensor3","MultiInValue":1,"Click":"single","click":"single","Endpoint":1,"LinkQuality":191}}}

that what I have by MQTT

**Comment by mdaskalov** [comment]
The problem here is that the StatelessProgrammableSwitch expects a numeric value and the "Click" value is string - "single". Currently there is no possibility to map values. 

You can try using MultiInValue as path instead.

**Comment by kiihmoo** [comment]
hey , 
i have tried that other value as well, 

        {
            "name": "tasmota_E32163",
            "mqttTopic": "tasmota_E32163",
            "zbBridgeDevices": [
                {
                    "addr": "0x00158D0008747677",
                    "type": "sensor",
                    "name": "Sensor3",
                    "sensorService": "StatelessProgrammableSwitch",
                    "sensorValuePath": "MultiInValue"
                }
            ],
            "platform": "TasmotaZbBridge"


but I there is nothing happening :-(

**Comment by mdaskalov** [comment]
I see. I will add value mapping with my next release.

**Comment by mdaskalov** [comment]
I also noticed, that the sensor characteristic is missing in your configuration. Please add `sensorCharacteristic` field:
```
    {
        "name": "tasmota_E32163",
        "mqttTopic": "tasmota_E32163",
        "zbBridgeDevices": [
            {
                "addr": "0x00158D0008747677",
                "type": "sensor",
                "name": "Sensor3",
                "sensorService": "StatelessProgrammableSwitch",
                "sensorCharacteristic": "ProgrammableSwitchEvent",
                "sensorValuePath": "MultiInValue"
            }
        ],
        "platform": "TasmotaZbBridge"

**Comment by kiihmoo** [comment]
cool, 

i have added that Characteristic

{
            "name": "tasmota_E32163",
            "mqttTopic": "tasmota_E32163",
            "zbBridgeDevices": [
                {
                    "addr": "0x00158D0008747677",
                    "type": "sensor",
                    "name": "Sensor3",
                    "sensorService": "StatelessProgrammableSwitch",
                    "sensorCharacteristic": "ProgrammableSwitchEvent",
                    "sensorValuePath": "MultiInValue"
                }
            ],
            "platform": "TasmotaZbBridge"
        }



Looking forward to your next release :-)

**Comment by kiihmoo** [comment]
Just in Case of being important

"MultiInValue" : 

On Click

{"ZbReceived":{"0xD394":{"Device":"0xD394","Name":"Sensor3","MultiInValue":1,"Click":"single","click":"single","Endpoint":1,"LinkQuality":175}}}

double click:

{"ZbReceived":{"0xD394":{"Device":"0xD394","Name":"Sensor3","MultiInValue":2,"Click":"double","click":"double","Endpoint":1,"LinkQuality":170}}}

Hold /

{"ZbReceived":{"0xD394":{"Device":"0xD394","Name":"Sensor3","MultiInValue":0,"Click":"hold","action":"hold","Endpoint":1,"LinkQuality":186}}}

Release

{"ZbReceived":{"0xD394":{"Device":"0xD394","Name":"Sensor3","MultiInValue":255,"Click":"release","action":"release","Endpoint":1,"LinkQuality":186}}}

**Comment by mdaskalov** [comment]
But it should work already, if you get different numeric values for MultiInValue. Open the button in the home app and look which functions get highlighted when you press it (once, twice, etc.). Look in the homebridge logs for any characteristic or value errors if not!

**Comment by kiihmoo** [comment]
Actually it does seem to work now, but there is some funny latency.

For example, 
i do press the button once, it does appear as a "one click" in MQTT

But for something to happen, I need to press it for something around a second , which in theory is not 1x Click but hold/release

**Comment by kiihmoo** [comment]
Is there a way to show this to you ? Maybe zoom or teams or something ?

**Comment by kiihmoo** [comment]
Hold / Release 255/0 are producing errors in the log file

witch Event': characteristic was supplied illegal value: number 255 exceeded maximum of 2

**Comment by mdaskalov** [comment]
Just released v1.8.0 with added mapping. Please try and give feedback (and a star)...

**Comment by kiihmoo** [comment]
Hey, I did not manage to use you mapping, nevertheless your plugin works great !
Can I leave you some tip anywhere? Thanks for your work !
